listenbrainz/troi/troi_bot.py

```diff
@@ -25,12 +27,18 @@ def run_post_recommendation_troi_bot():
 
 def run_daily_jams_troi_bot():
     """ Top level function called hourly to generate daily jams playlists for users """
-
     # Now generate daily jams (and other in the future) for users who follow troi bot
     users = get_users_for_daily_jams()
+    # TODO: Add endpoint to configure auto-export to spotify and do check scopes,
```

Comment:
To discuss, what should be done when a user disconnects their spotify account? Should we clear the export to spotify preference or should this code be made more intelligent to handle that case? Maybe a mix?

Reply:
Handling spotify submission failue gracefully will also need changes in troi.

Reply:
Discussed and settled on latter.